bitnami/kafka/Chart.yaml

```diff
@@ -24,4 +24,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 12.0.0
+version: 12.1.0
```

Comment:
I have resolved some conflicts on your behalf and increased the minor version instead of the patch as this PR is introducing new parameters.